src/flaskr/templates/base.html

```diff
@@ -9,6 +9,11 @@
 </head>
 <body>
     <header class="top-header">
+        <style>
+            body {
+                font-family: Arial, sans-serif;
+            }
```

Comment:
```suggestion
            }
            h2 {
                 text-align: center;
           }
```

Reply:
![Screenshot 2024-03-24 182204](https://github.com/Birdbh/Engicoders/assets/77585779/7212fb76-5161-4336-8bb2-e00793f25063)

I'm trying to streamline this into style.css, so that we have less bloat in the actual HTML files. Not sure if there's something in the formatting of the HTML files, but I'm having issues getting my changes in style.css to show up on the flask application when updated.